src/ResourceManager/KeyVault/Commands.KeyVault/Commands/ImportAzureKeyVaultCertificate.cs

```diff
@@ -105,8 +101,11 @@ public class ImportAzureKeyVaultCertificate : KeyVaultCmdletBase
         /// Certificate Collection
         /// </summary>
         [Parameter(Mandatory = true,
+                   Position = 2,
+                   ValueFromPipeline = true,
                    ParameterSetName = ImportWithPrivateKeyFromCollectionParameterSet,
                    HelpMessage = "Specifies the certificate collection to add to key vault.")]
+        [Alias("InputObject")]
```

Comment:
@maddieclayton any reason we don't want to call this parameter `InputObject` with an alias to `CertificateCollection` like we do for all other cmdlets?

Reply:
Deleting alias as it doesn't really make sense in this context.